core-ui/src/components/Predefined/Details/Sidecars.details.js

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { EMPTY_TEXT_PLACEHOLDER } from 'react-shared';
+import { LayoutPanel } from 'fundamental-react';
+import { LayoutPanelRow } from 'shared/components/LayoutPanelRow/LayoutPanelRow';
+import { WorkloadSelector } from 'shared/WorkloadSelector/WorkloadSelector';
+
+const IstioListeners = sidecar => {
+  const { t } = useTranslation();
+  return (
+    <>
+      <ListenerView
+        listener={sidecar.spec?.egress}
+        type={t('sidecars.headers.egress')}
+        key="egress-listener"
+        isEgress
+      />
+      <ListenerView
+        listener={sidecar.spec?.ingress}
+        type={t('sidecars.headers.ingress')}
+        key="ingress-listener"
+      />
+    </>
+  );
+};
+
+const Hosts = ({ hosts }) => {
+  return (
+    <ul>
+      {hosts?.map((host, i) => (
+        <li key={i}>{host}</li>
+      ))}
+    </ul>
+  );
+};
+
+const Port = ({ port }) => {
```

Comment:
Displaying port details does look a little bit funky. Can we make it more readable
<img width="853" alt="Screen Shot 2022-02-16 at 08 05 57" src="https://user-images.githubusercontent.com/20790348/154213619-1e28f67d-c847-4677-b5e2-a6a2c49d8dd8.png">
especially when there might be multiple egress objects (like in sample Sidecar in ak namespace)